src/Components/Header/Header.js

```diff
@@ -1,14 +1,20 @@
+import { useNavigate } from 'react-router-dom';
 import voteImage from '../../Assets/vote.png';
 import './Header.css';
 
 const Header = ()  => {
+  let navigate = useNavigate();
+
   return (
     <div className="header-container">
       <img
         className="vote-image"
         src={voteImage}
         alt="vote"/>
-      <p className="app-name">My Voice, My Vote</p>
+      <p className="app-name" 
+        onClick={() => navigate("/")}>
+        My Voice, My Vote
+      </p>
```

Comment:
As discussed, this `p` tag was updated to an `h1` with a `font-weight: 200` rule on my branch.

Reply:
The header is now a Link component.